packages/core/src/test/codewhisperer/commands/transformByQ.test.ts

```diff
@@ -6,6 +6,7 @@
 import assert from 'assert'
 import * as vscode from 'vscode'
 import * as fs from 'fs-extra'
+import * as os from 'os'
 import * as sinon from 'sinon'
 import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
```

Comment:
tests should use `createTestWorkspaceFolder`
```suggestion
import { createTestWorkspaceFolder } from '...'
```